beacon-chain/core/epoch/epoch_processing_test.go

```diff
@@ -593,19 +541,6 @@ func TestBaseReward_AccurateRewards(t *testing.T) {
 	}
 }
 
-func TestProcessJustificationAndFinalization_LessThan2ndEpoch(t *testing.T) {
```

Comment:
why is this deleted ?

Reply:
Because this condition was removed. https://github.com/prysmaticlabs/prysm/pull/2896/files/e46c6b14a888fd8ed91fcefe8818102c1a4f1fab#diff-d00788a30b2b85085fd4cb09a50a7f75L171